ci-operator/step-registry/baremetal/lab/agent/conf/static/baremetal-lab-agent-conf-static-commands.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+RENDEZVOUS_IP="$(yq -r e -o=j -I=0 ".[0].ip" "${SHARED_DIR}/hosts.yaml")"
+
+cat > "${SHARED_DIR}/agent-config.yaml" <<EOF
+apiVersion: v1beta1
+kind: AgentConfig
+rendezvousIP: ${RENDEZVOUS_IP}
+additionalNTPSources:
+- ${AUX_HOST}
+hosts: []
+EOF
+
+# shellcheck disable=SC2154
+for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
+  # shellcheck disable=SC1090
+  . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  ADAPTED_YAML="
+  hostname: ${name}
+  role: ${name%%-[0-9]*}
+  rootDeviceHints:
+    ${root_device:+deviceName: ${root_device}}
+    ${root_dev_hctl:+hctl: ${root_dev_hctl}}
+  interfaces:
+  - macAddress: ${mac}
+    name: ${baremetal_iface}
+  networkConfig:
+    interfaces:
+    - name: ${baremetal_iface}
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: ${ipv4_enabled}
+        dhcp: false
+        address:
+            - ip: ${ip}
+              prefix-length: 23
```

Comment:
this must be set correctly. 

use ` ${INTERNAL_NET_CIDR##*/}`

Reply:
like this " prefix-length: ${INTERNAL_NET_CIDR##*/} " ?

Reply:
changed